CHANGELOG.md

```diff
@@ -17,6 +17,9 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.1.11] - 19-05-23
```

Comment:
Fix the date here.
```suggestion
## [6.1.11] - 22-05-23
```

Reply:
As we are adding functionality in a compatible way, I assume this should be 6.2.0?

Reply:
@SunilVenekarKrishnamoorthy